tools/autograd/templates/Functions.cpp

```diff
@@ -1,4 +1,5 @@
 #include "torch/csrc/autograd/FunctionsManual.h"
+#include "torch/csrc/autograd/compiled_autograd.h"
```

Comment:
Why does this needs to know about compiled_autograd?

Reply:
To get the definitions of the visitor objects.

Reply:
Can you split it up in a separate file that we depend on here?

Reply:
What do you mean?  `compiled_autograd.h` is already just the visitor objects.  The other stuff is in `python_compiled_autograd.cpp`.